addons/survey/views/survey_templates.xml

```diff
@@ -300,6 +300,12 @@
         </div>
     </template>
 
+    <template id="question_answer_image" name="Question answer: image">
+        <div t-if="question.answer_image and label.image" class="d-flex my-2 justify-content-center" style="height: 80%">
```

Comment:
Never inline-style if you can help it.
Use boostrap classes, and if you have to, add specific classes and css in the scss file.

Reply:
done